Gemfile

```diff
@@ -97,21 +97,21 @@ gem "handlebars_assets", "0.23.9"
 gem "jquery-rails",      "4.5.0"
 gem "jquery-ui-rails",   "5.0.5"
 gem "js_image_paths",    "0.2.0"
-gem "js-routes",         "2.2.4"
+gem "js-routes",         "2.2.5"
 
 # Localization
 
 gem "http_accept_language", "2.1.1"
-gem "rails-i18n",           "6.0.0"
+gem "rails-i18n",           "7.0.7"
 
 # Map
-gem "leaflet-rails",       "1.7.0"
+gem "leaflet-rails", "1.9.3"
```

Comment:
LOL no ... I removed the unnecessary spacing ... but I guess that's because the bot resets the Gemfile to `develop`, where the spaces appear again. :man_shrugging: